draft-enghardt-panrg-path-properties.md

```diff
@@ -63,6 +63,30 @@ There are exceptions such as dynamic wireless access properties, but these do no
 This document addresses the first of the questions
 in Path-Aware Networking {{I-D.irtf-panrg-questions}}, which is a product of the PANRG in the IRTF.
 
+# Terminology
+
+Path element:
+: A path element is a device (including the endpoints), or link used to connect two endpoints and transmit information on a specific layer.
+Path elements may exist on multiple layers (e.g., the endpoint corresponds to a path element on every layer), may be hidden on higher layers (e.g., a layer 2 switch in the local network), or a path element may be an aggregation of several path elements on a lower layer (e.g., the link connecting the endpoints on the transport layer being an aggregation of all network layer path elements).
+
+Path:
+: A path is defined as an ordered set of path elements at the network layer that can be traversed by a packet at specific points in time.
+For the sake of simplicity, we use the term packet, typically used at the network layer, to describe bit strings transmitted on any layer.
+
+Flow:
+: Several packets traversing the same path elements at specific points in time, can be combined into a flow (e.g., all packets sent within a UDP session which traverse the same path elements).
```

Comment:
I would keep the definition on a part of the path instead of on a complete path, in order to be able to for example specify: "median one-way delay outside of local network", which is the median delay of the flow after the second hop to the endpoint.

Reply:
Fine with me, it does seem more useful to also have it apply on part of the path.